src/unstructured_client/_hooks/custom/pdf_utils.py

```diff
@@ -56,25 +56,20 @@ def get_pdf_pages(
 def is_pdf(file: shared.Files) -> bool:
     """Checks if the given file is a PDF.
 
-    First it checks the file extension and if it is equal to `.pdf`, then
-    it tries to read that file. If there is no error then we assume it is a proper PDF.
+    Tries to read that file. If there is no error then we assume it is a proper PDF.
 
     Args:
         file: The file to be checked.
 
     Returns:
         True if the file is a PDF, False otherwise.
     """
-    if not file.file_name.endswith(".pdf"):
-        logger.info("Given file doesn't have '.pdf' extension, so splitting is not enabled.")
-        return False
 
     try:
         content = cast(bytes, file.content)
         PdfReader(io.BytesIO(content), strict=True)
-    except (PdfReadError, UnicodeDecodeError) as exc:
-        logger.error(exc)
-        logger.warning("The file does not appear to be a valid PDF.")
+    except (PdfReadError, UnicodeDecodeError):
+        logger.info("Loading PDF failed, so splitting is not enabled.")
```

Comment:
Sorry to be nitpicky, but I would skip logging anything in this path. Since the user will see the word failed and think something's wrong with their non PDF file. Plus we already have some noisy logs to clean up